src/components/molecules/FormSection/FormSection.module.scss

```diff
@@ -0,0 +1,5 @@
+.border {
```

Comment:
some people are using css files and some people are defining any extra styles needed in the tsx files - do we want to have an agreed standard?

Reply:
Yeah I wasn't too sure about this since it is pretty straightforward and inline style makes sense. Happy to change if that's the direction we want to go.

Reply:
Lets use CSS files.

Reply:
Do we need a tech debt ticket to standardise it?

Reply:
Added to tidy up for now https://nhsd-jira.digital.nhs.uk/browse/CCM-6950